src/api/spec/controllers/webui/staging/workflows_controller_spec.rb

```diff
@@ -46,6 +47,8 @@
       it { expect(response).to redirect_to(staging_workflow_path(project.staging)) }
       it { expect(subject.managers_group).to eq(managers_group) }
       it { expect(flash[:success]).not_to be_nil }
+      it { expect(reviewer.group).to eq(managers_group) }
```

Comment:
That seems to be a duplication of the query you run in the let in line 38.

Reply:
Changed as you proposed in the previous comment